spec/amp-cache-guidelines.md

```diff
@@ -88,6 +88,14 @@ Accepted MIME types for fonts include the following prefixes:
 |  binary/octet-stream   |
 |  text/plain (not recommended)   |
 
+## Guidelines: Adding a new cache to the AMP ecosystem
+
+* Contact relevant publishers to make sure they update their CORS policies to include your origin.
```

Comment:
@lannka Done as per:
* Contact relevant publishers to make sure they update their CORS policies to include your origin.
  * For further information please refer to the [CORS Security in AMP Guidelines](https://github.com/ampproject/amphtml/blob/master/spec/amp-cors-requests.md#cors-security-in-amp).